lib/roo/xls/excel_2003_xml.rb

```diff
@@ -18,8 +18,10 @@ def initialize(filename, options = {})
       unless File.file?(@filename)
         fail IOError, "file #{@filename} does not exist"
       end
-      @doc = load_xml(@filename)
+      @doc = ::Roo::Utils.load_xml(@filename)
     end
+    namespace = @doc.namespaces.select{|xmlns, urn| urn == 'urn:schemas-microsoft-com:office:spreadsheet'}.keys.last
+    @namespace = namespace.nil? || namespace.empty? ? 'ss' : namespace.split(':').last
```

Comment:
You are right. I don't know the XML specs (and I doubt that M$ cares anyway), but to me there has to be EXACTLY ONE namespace given with the value `urn:schemas-microsoft-com:office:spreadsheet`. You could take the first one, the last one or anyone. It's just a matter of taste?